target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -81,13 +85,20 @@ mediatek_setup_macs()
 	local label_mac=""
 
 	case $board in
+	asus,rt-ax59u|\
 	asus,tuf-ax4200|\
 	asus,tuf-ax6000)
 		CI_UBIPART="UBI_DEV"
 		addr=$(mtd_get_mac_binary_ubi "Factory" 0x4)
 		wan_mac="${addr}"
 		lan_mac="${addr}"
 		;;
+	asus,rt-ax59u-ubootmod)
+		CI_UBIPART="UBI_DEV"
+		addr=$(mtd_get_mac_binary "Factory" 0x4)
+		wan_mac="${addr}"
+		lan_mac="${addr}"
+		;;
```

Comment:
Do you really need this? You have `macaddr_factory_4` in dts and can assign the lan and wan mac also in dts.

Reply:
I think it's the easiest and and at the same time the best choice.